contrib/bloop/src/mill/contrib/bloop/BloopImpl.scala

```diff
@@ -133,14 +137,19 @@ class BloopImpl(evs: () => Seq[Evaluator], wd: os.Path) extends ExternalModule {
       !module.asInstanceOf[outer.Module].skipBloop
     else true
 
+  private def deferredAllSources(m: JavaModule): T[Seq[PathRef]] = Task {
+    m.sources() ++ m.generatedSources() ++ m.allDeferredSourceRoots()
```

Comment:
Keeps the current (0.12.x) behaviour of `generatedSources` being run to produce IDE config, in order to avoid breaking the ecosystem